app/helpers/wishlists_helper.rb

```diff
@@ -0,0 +1,2 @@
+module WishlistsHelper
```

Comment:
the same I can do with wishlists.js and scss because right now we don't need them.